hardware/simulation/testbench/iob-cache_tb.vh

```diff
@@ -28,7 +28,7 @@
 `define CTRL
 
 // print VCD for GTKWave (necessary)
-`define VCD
+//`define VCD
```

Comment:
No useless comments please -- just delete this, I can see the changes and reply back if I disagree 

Please fix and push -- the PR will update automatically, no need to issue a new one